css/symbiota/taxa/index.css

```diff
@@ -1,340 +1,337 @@
-#innertable{
-	width: 95%;
-	display: table;
-	margin-top: 15px;
-	margin-left: auto;
-	margin-right: auto;
+#innertable {
+  width: 95%;
+  display: table;
+  margin-top: 15px;
+  margin-left: auto;
+  margin-right: auto;
 }
 
-#toprow{
-	width: 100%;
-	display: table-row;
-	clear: both;
+#toprow {
+  width: 100%;
+  display: table-row;
+  clear: both;
 }
 
-#middlerow{
-	width: 100%;
-	display: table;
-	clear: both;
+#middlerow {
+  width: 100%;
+  display: table;
+  clear: both;
 }
 
-#leftcolumn{
-	display: table-cell;
+#leftcolumn {
+  display: table-cell;
 }
 
-#rightcolumn{
-	display: table-cell;
-	vertical-align:	top;
-	width: 525px;
+#rightcolumn {
+  display: table-cell;
+  vertical-align: top;
+  width: 525px;
 }
 
-#bottomrow{
-	width: 100%;
-	display: table-row;
-	clear: both;
+#bottomrow {
+  width: 100%;
+  display: table-row;
+  clear: both;
 }
 
-#footerrow{
-	width: 100%;
-	display: block;
-	margin-top: 10px;
-	text-align: center;
-	clear: both;
+#footerrow {
+  width: 100%;
+  display: block;
+  margin-top: 10px;
+  text-align: center;
+  clear: both;
 }
 
-#innerDiv{
-	width: 95%;
-	display: table;
-	margin-top: 15px;
-	margin-left: auto;
-	margin-right: auto;
+#innerDiv {
+  width: 95%;
+  display: table;
+  margin-top: 15px;
+  margin-left: auto;
+  margin-right: auto;
 }
 
-#leftPanel{
-	width: 30%;
-	float: left;
+#leftPanel {
+  width: 30%;
+  float: left;
 }
 
-#rightPanel{
-	width: 70%;
-	float: left;
+#rightPanel {
+  width: 70%;
+  float: left;
 }
 
 @media screen and (max-width: 800px) {
-	#leftPanel #rightPanel {
-		width: 100%; 
-	}
+  #leftPanel #rightPanel {
+    width: 100%;
+  }
 }
 
-fieldset{
-	margin: 10px;
-	padding: 10px
+fieldset {
+  margin: 10px;
+  padding: 10px;
 }
 
-fieldset legend{
-	font-weight: bold;
+fieldset legend {
+  font-weight: bold;
 }
 
-#scinameDiv{
-	font-size: 1.3em;
-	font-weight: bold; 
-	margin-bottom: 10px;
+#scinameDiv {
+  font-size: 1.3em;
+  font-weight: bold;
+  margin-bottom: 10px;
 }
 
-#sciname{ 
-	color: #990000; 
-	font-style: italic;
+#sciname {
```

Comment:
Removed italic from here. Everything else in this file is an autoformatting change.